test/claims-status/utils/helpers.unit.spec.js

```diff
@@ -469,4 +470,16 @@ describe('Disability benefits helpers: ', () => {
       expect(contents.description).to.equal('Your current appeal status is unknown at this time');
     });
   });
+
+  describe.only('getNextEvents', () => {
+    it('returns an array of next event objects', () => {
+      const type = STATUS_TYPES.nod;
+      const nextEvents = getNextEvents(type);
+      expect(nextEvents.length).to.equal(2);
```

Comment:
It isn't immediately clear why this is 2. Could you put a comment in here explaining why?